Mathlib/Order/Filter/Basic.lean

```diff
@@ -963,7 +962,7 @@ theorem EventuallyEq.comp₂ {δ} {f f' : α → β} {g g' : α → γ} {l} (Hf
 
 @[to_additive]
 theorem EventuallyEq.mul [Mul β] {f f' g g' : α → β} {l : Filter α} (h : f =ᶠ[l] g)
-    (h' : f' =ᶠ[l] g') : (fun x => f x * f' x) =ᶠ[l] fun x => g x * g' x :=
+    (h' : f' =ᶠ[l] g') : f * f' =ᶠ[l] g * g' :=
```

Comment:
I think the decision here was to duplicate the API, keeping the old lemma with the name `fun_mul`. Maybe @sgouezel has opinions, as they wrote #26355.